### ISSUE TITLE
Transfer CDL Tiles to Penn State

### ISSUE THREAD

**Issue by rajadain** [project-icp/bee-pollinator-app#620]
Transfer the contents of s3://tiles.us-east-1.azavea.com/cdl-reclass-2017-gdal to Penn State's Sharepoint server.

![image.png](https://images.zenhubusercontent.com/5c9cd6db2250f9495b572d5b/9381c8e1-8344-40d2-bf74-595bdb43676c)

**Comment by rajadain** [comment]
I downloaded all the files from S3 using `aws s3 sync` and tried uploading to the SharePoint drive they pointed us to, but the browser uploads failed repeatedly for the size and number of files.

I ended up using `rclone` to upload a `.tgz` archive to Google Drive, sent them a link to it, and got confirmation that they downloaded it. I have since deleted the file from Google Drive to reclaim space.

I have moved the file to the project folder in the fileshare for future use:

```
smb://fileshare/projects/PennStateUniversity_USDABeekeepers/data/cdl-reclass-2017-gdal.tgz
```
